compiler/noirc_evaluator/src/ssa/acir_gen/mod.rs

```diff
@@ -1730,12 +1730,15 @@ impl<'a> Context<'a> {
         let return_acir_vars = self.flatten_value_list(return_values, dfg)?;
         let mut warnings = Vec::new();
         for acir_var in return_acir_vars {
```

Comment:
```suggestion
        for (acir_var, is_databus) in return_acir_vars {
```
Could we do this for clarity around the tuple?